sys/kern/syscalls.master

```diff
@@ -1507,7 +1507,21 @@
 		    uint32_t key
 		);
 	}
-260-271	AUE_NULL	RESERVED
+260	AUE_NULL	STD|CAPENABLED	{
```

Comment:
I would maybe include the stub syscalls along with the syscalls.master change as a single commit so that there's just the 1 commit window where it doesn't build.  This more closely matches upstream where everything but the regen is committed first and then the regen as a followup.